lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -224,6 +224,55 @@ function gutenberg_get_global_settings( $path = array(), $context = array() ) {
 		$origin = 'theme';
 	}
 
-	$settings = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data( $origin )->get_settings();
+	$cache_group = 'theme_json';
+	$cache_key   = 'gutenberg_get_global_settings_' . $origin;
+	$settings    = wp_cache_get( $cache_key, $cache_group );
+
+	if (
+		false === $settings ||
+		( defined( 'WP_DEBUG' ) && WP_DEBUG )
```

Comment:
```suggestion
		WP_DEBUG
```

WP_DEBUG is always defined.

Reply:
Let's not load the cache, if we are not doing to use it.